lib/html.js

```diff
@@ -278,7 +279,7 @@ function HtmlRenderer(options){
         // set to "<br />" to make them hard breaks
         // set to " " if you want to ignore line wrapping in source
         escape: escapeXml,
-        options: options || {},
+        options: objectAssign({ sanitize: function() { return '<!-- raw HTML omitted -->'; }, isUrlSafe: isUrlSafe }, options),
```

Comment:
One could instead do
```
options.sanitize = ...
options.isUrlSafe = ...
```
and then as before
```
options: options || {}
```
which would avoid the need for all that `objectAssign` business.  What's to be gained from doing it this way instead?